changelog/changelog.py

```diff
@@ -324,8 +336,17 @@ def main():
         'generator': default_changelog_generator,
     },
     'spinalcordtoolbox': {
-        'labels': ['feature', 'documentation-internal', 'CI', 'bug', 'installation', 'documentation', 'enhancement',
-                   'refactoring', 'git/github'],
+        'labels': [
+            'feature',
+            'enhancement',
+            'bug',
+            'installation',
+            'documentation',
+            'documentation-internal',
+            'refactoring',
+            'CI',
+            'git/github',
+        ],
         'generator': sct_changelog_generator,
     },
     'ivadomed': {
```

Comment:
While we're at it, should we also rewrite the ivadomed/ADS labels into one-line-per-label?

Reply:
Sure, will do.

Reply:
Done in [`6ec3e0b`](https://github.com/neuropoly/changelog/pull/33/commits/6ec3e0b95d8fd4d17c28d129ff61dc6cabf0f939).